clock/clock.go

```diff
@@ -108,6 +117,13 @@ func (RealClock) NewTimer(d time.Duration) Timer {
 	}
 }
 
+// NewDeadlineTimer is the same as time.NewTimer(ts.Sub(time.Now()))
+func (RealClock) NewDeadlineTimer(ts time.Time) Timer {
+	return &realTimer{
+		timer: time.NewTimer(ts.Sub(time.Now())),
```

Comment:
`ts.Sub(time.Now)` can be written as `time.Until(ts)`

Reply:
Fixed via 50647950f2abc950ddc3c2e245982966a51d6fb1